recipes/librmm/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "librmm" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: eddd5b35b016d665eb4e8ea2dd31497d913b3e1eb831124e4cb27cba747267a0
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.18
+    - make
+  run:
```

Comment:
We also need to include thrust here as we leak the thrust dependency: https://github.com/rapidsai/rmm/blob/branch-0.20/cmake/Modules/RMM_thirdparty.cmake

Reply:
Added in next commit. I used `>=1.9.0` based on the min version in the linked file.

Reply:
@kkraus14 Does this mean all downstream packages of librmm are forced to use Conda-Forge's Thrust/CUB?